src/main/java/com/wolox/training/services/BookService.java

```diff
@@ -0,0 +1,76 @@
+package com.wolox.training.services;
+import com.wolox.training.exceptions.BadRequestException;
+import com.wolox.training.exceptions.NotFoundException;
+import com.wolox.training.exceptions.ServerErrorException;
+import com.wolox.training.repositories.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.wolox.training.models.Book;
+
+import java.util.List;
+
+@Service
+public class BookService {
```

Comment:
I liked that you use services to encapsulate logic that should not be in the controller, but in this method validations are made of the attributes of objects of the book class that should be done with annotation at  model level.